packages/electrode-archetype-opt-inferno/optional-check.js

```diff
@@ -13,29 +13,33 @@ function findAppDir() {
 }
 
 const appDir = findAppDir();
+const myPkg = require("./package.json");
```

Comment:
assumes the current directory has a package.json.

Reply:
not assumes, ensure.

Reply:
then maybe test the cwd to ensure it's package root so we can display a descriptive message.

this is going to say "can't read package.json".. which isn't very informative.